### PR TITLE
Filter do not merge

### PR DESCRIPTION
**Pull request by EmmaOnThursday**
Don't auto-merge PRs with the "do not merge" label. 

also removed some commented code, and renamed a thing.

### CODE REVIEW

File: src/github/github.js [hunk 1]
```diff
@@ -212,14 +204,20 @@ const textMatchesString = string => text => text.includes(string);
 const textMatchesAnyString = stringSet => text => stringSet.some(s => text.includes(s));
 
 const textMatchesJankIndex = textMatchesString('jank:');
-const textMatchesShippit = textMatchesAnyString(shippedEmojis);
+const textMatchesShipit = textMatchesAnyString(shippedEmojis, true);
```
Comment (er9781) [inline src/github/github.js]: what is the `true`?

Comment (er9781) [reply src/github/github.js]: also with the typo correction, I believe we'll need to change the caller too ;)

File: src/github/github.js [hunk 2]
```diff
@@ -230,22 +235,27 @@ const textMatchesJankIndex = textMatchesString('jank:');
...
-  const prs = (await getOpenPrs(pullReqs)).filter(pr => {
+const prsToTriggered = async (textFilter, pullReqs, filterByLabelName) => {
+  let prs = (await getOpenPrs(pullReqs)).filter(pr => {
     return textFilter(pr.body) ||
         pr.comments.nodes.map(c => c.body).some(textFilter);
   });
+  if (filterByLabelName) {
+    prs = prs.filter(
+        pr => {return pr.labels.nodes.map(l => l.name.toLowerCase())
+                   .includes('do not merge')})
```
Comment (er9781) [inline src/github/github.js]: ⛳️ `pr => pr.labels.nodes.some(l => l.name.toLowerCase() === 'do not merge')`

File: src/github/github.js [hunk 3]
```diff
@@ -230,22 +235,27 @@ const textMatchesJankIndex = textMatchesString('jank:');
 const textMatchesShippit = textMatchesAnyString(shippedEmojis);
 const textMatchesUpdate = textMatchesAnyString(updateMeEmojis);
 
-const prsToTriggered = async (textFilter, pullReqs) => {
-  const prs = (await getOpenPrs(pullReqs)).filter(pr => {
+const prsToTriggered = async (textFilter, pullReqs, filterByLabelName) => {
```
Comment (er9781) [inline src/github/github.js]: prefer to see `filterByLabelName = false` and then only passing it in from callers when it's required.

**Comment by er9781** [comment]
Also would be good to have prettier run with config to not change the whole file :). nbd tho cus my editor will just reformat it next time I'm working on it.
```
  "printWidth": 120
```

**Comment by EmmaOnThursday** [comment]
> Also would be good to have prettier run with config to not change the whole file :). nbd tho cus my editor will just reformat it next time I'm working on it.
> 
> ```
>   "printWidth": 120
> ```

weirdly this is already my VSCode setting but it is definitely wrapping at 80 char. I turned off all automatic formatting so at least it won't conflict with yours - you may need to reformat a few lines next time you edit github.js

**Pull request merged**